src/XML/src/FMI3/fmi3_xml_variable.c

```diff
@@ -174,6 +174,11 @@ fmi3_boolean_t fmi3_xml_get_canHandleMultipleSetPerTimeInstant(fmi3_xml_variable
     return (fmi3_boolean_t)v->canHandleMultipleSetPerTimeInstant;
 }
 
+fmi3_boolean_t fmi3_xml_is_clocked(fmi3_xml_variable_t* v) {
```

Comment:
New auxiliary function.

Reply:
I suggest `fmi3_xml_variable_is_clocked` similar to `fmi3_xml_variable_is_array`. I know some other functions don't prefix with variable (but they should... but they match import API so not really worth changing).

Return type should also be `bool` or `int`. We only use the FMI types if the object corresponds to something immediately defined in the XML.

Reply:
Took the naming suggestion and changed to int.